### PR TITLE
*: enforce camel case variable names

### PR DESCRIPTION
**Pull request by Matthalp**
This replaces variables in snake case with the corresponding camel case.

### CODE REVIEW

**Comment by Matthalp** [comment]
Please advise what the correct commit message prefix should be. It was unclear how to handle a change that superficially affects so many packages.

File: core/vm/runtime/runtime_test.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func BenchmarkCall(b *testing.B) {
 		}
 	}
 }
-func benchmarkEVM_Create(bench *testing.B, code string) {
+func benchmarkevmCreate(bench *testing.B, code string) {
```
Comment (karalabe) [inline core/vm/runtime/runtime_test.go]: I think `benchmarkEVMCreate` would be the correct capitalization.

Comment (Matthalp) [reply core/vm/runtime/runtime_test.go]: Thanks for catching this.

**Comment by Matthalp** [comment]
Closing as the changes have been divided into more granular CLs:
light: #19054
les: #19055
ethdb: #19056
eth/tracers: #19057
core: #19058
console, internal/*: #19059
cmd, swarm: #19060

**Pull request closed without merging**